src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/PerformanceFormHandler.php

```diff
@@ -77,19 +77,11 @@ public function getForm()
             ->add('add_memcache_server', MemcacheServerType::class)
             ->setData($this->formDataProvider->getData());
 
-        $this->hookDispatcher->dispatchWithParameters('displayPerformancePageForm', ['form_builder' => &$formBuilder]);
```

Comment:
Oh ? 😮 

I understand that `$this->formDataProvider->setData($data);` will fire some hooks that provide the right extension endpoints, however removing these (useless) hooks here is a BC break, unrelated to PHPStan introduction ?

Reply:
Could you check a new time ? There is no property `hookDispatcher` !

Reply:
Hoho 😄 indeed that class is competely broken !

Good thing it's deprecated
```php
@deprecated since 1.7.4.0, to be removed in the next major
```

So my review is 👍